docs/executor-custom.md

```diff
@@ -101,11 +162,209 @@ echo "http://google.pl" | kubectl testkube tests create --name example-google-te
 kubectl testkube tests run example-google-test
 ```
 
-That's all for the most basic executor example, you can look our internal projects for more examples
-and details how it's implemented:
+That's all for the most basic executor example, you can look our internal projects for more examples and details how it's implemented:
```

Comment:
What we have shown is a most basic executor example. Please visit our internal projects for more examples and the details on implementation: